pages/instructorCourseAdminGrading/instructorCourseAdminGrading.ejs

```diff
@@ -2,9 +2,6 @@
 <html lang="en">
     <head>
         <%- include('../partials/head', {pageTitle: 'External Grading'}); %>
-        <script src="/javascripts/lodash.min.js"></script>
-        <script src="/javascripts/d3.min.js"></script>
-        <script src="/localscripts/histmini.js"></script>
```

Comment:
Wow, so many pages including so much unnecessary JS!

Reply:
Right! Blind copy-pasting coming back to bite us 😛